gridspeccer/cli.py

```diff
@@ -29,6 +29,12 @@ def plot():
             osp.dirname(osp.abspath(__file__)), "defaults", "matplotlibrc"
         ),
     )
+    parser.add_argument(
+        "--loglevel",
+        help="Display more or less info (values: DEBUG INFO WARNING ERROR)",
+        type=str,
```

Comment:
Choices should be limited: https://docs.python.org/3/library/argparse.html#choices

Reply:
interesting, learned something